datetime_tz/update_win32tz_map.py

```diff
@@ -120,9 +121,13 @@ def update_stored_win32tz_map():
       source_hash))
 
   map_file.write("win32timezones = {\n")
-  for win32_name, olson_name, comment in map_zones:
-    map_file.write("  %r: %r, # %s\n" % (
-        win32_name, olson_name, comment or ""))
+  for win32_name, territory, olson_name, comment in map_zones:
+    if territory == '001':
+      map_file.write("  %r: %r, # %s\n" % (
+          str(win32_name), str(olson_name), comment or ""))
+    else:
+      map_file.write("  %r: %r, # %s\n" % (
```

Comment:
It may be clearer to say `"  (%r, %r): %r, # %s\n"` rather than passing in the tuple - just for readability